### PR TITLE
LPS-93377 Update CKEditor to v4.11.3

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test

**Comment by julien** [comment]
/cc @john-co 
This PR updates CKEditor to v4.11.3 (with no patches applied)

Could you please see if these patches are still required when you have the time?

[LPS-73165](https://issues.liferay.com/browse/LPS-73165)
[LPS-74837](https://issues.liferay.com/browse/LPS-74837)
[LPS-75876](https://issues.liferay.com/browse/LPS-75876)
[LPS-85326](https://issues.liferay.com/browse/LPS-85326)
[LPS-86456](https://issues.liferay.com/browse/LPS-86456)
[LPS-88974](https://issues.liferay.com/browse/LPS-88974)
[LPS-89596](https://issues.liferay.com/browse/LPS-89596)
[LPS-90595](https://issues.liferay.com/browse/LPS-90595)

Thanks!

File: modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public void populateConfigJSONObject(
 		jsonObject.put(
 			"removePlugins",
 			"contextmenu,elementspath,image,link,liststyle,resize,tabletools," +
-				"toolbar");
```
Comment (jbalsas) [inline modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java]: sort this, please 🙏

Comment (julien) [reply modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java]: Thanks

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/70820. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**